docs/integrations/sources/amazon-ads.md

```diff
@@ -103,6 +109,7 @@ Information about expected report generation waiting time you may find [here](ht
 
 | Version | Date       | Pull Request                                             | Subject                                                                                                         |
 |:--------|:-----------|:---------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------|
+| 3.1.1   | 2023-08-16 | [29233](https://github.com/airbytehq/airbyte/pull/29233) | Add filter for Marketplace IDs |
```

Comment:
Would you mind adding extra spaces at the end of each row to ensure the table aligns properly? This is not a requirement, just a minor suggestion for improved readability in raw format.